core/market/decentralized/tests/test_initial_proposal.rs

```diff
@@ -1,418 +1,411 @@
-mod utils;
```

Comment:
This is good change. Maybe compiler will stop reporting duplicated errors

Reply:
it was my intention